package.nls.json

```diff
@@ -1,33 +1,33 @@
 {
     "extension.description": "An Azure Functions extension for Visual Studio Code.",
     "azFunc.refresh": "Refresh",
-    "azFunc.loadMore": "Load More...",
-    "azFunc.createNewProject": "Create New Project",
-    "azFunc.initProjectForVSCode": "Initialize project for use with VS Code",
-    "azFunc.createFunction": "Create Function",
-    "azFunc.createFunctionApp": "Create Function App in Azure",
-    "azFunc.openInPortal": "Open In Portal",
+    "azFunc.loadMore": "Load More",
```

Comment:
I don't think Delete needs it.  IMHO a confirmation dialog doesn't really count.

Reply:
Yeah I was iffy on this as well. However I just went looking for more guidance and the general Microsoft docs explicitly say a confirmation dialog counts:
> Indicate a command that needs additional information (including a confirmation) by adding an ellipsis at the end of the label.

https://docs.microsoft.com/windows/desktop/uxguide/cmd-menus#using-ellipses